packages/@react-types/breadcrumbs/src/index.d.ts

```diff
@@ -26,25 +26,13 @@ export interface BreadcrumbItemProps extends PressEvents {
   isHeading?: boolean,
   /** Whether the breadcrumb item is disabled. */
```

Comment:
Ugh, I had to bring this back instead of relying on `elementType` since BreadcrumbItem does this:
```
  let {breadcrumbItemProps} = useBreadcrumbItem({
    ...props,
    elementType: typeof children === 'string' ? 'span' : 'a'
  });
```
The problem is that I can't think of a good way to distinguish the children type between a `Heading` element (where I would be able to pass `elementType: 'h1'` or something) and any other component that might be inside the Breadcrumb's `Item` (it could be <Text> or something else).